server/apps/users/helpers.py

```diff
@@ -53,3 +54,25 @@ def get_user_profile(user):
     except UserProfile.DoesNotExist:
         uo = None
     return uo
+
+def delete_user(user, delete_oh_data):
+    """
+    Deletes the user and all data associated with it.
+
+    Args:
+        delete_oh_data: True if stories on OpenHumans should also be deleted
+
+    Returns:
+        None
+    """
+    ohmember = user.openhumansmember
+
+    # Delete the stories from the OpenHumans database
+    if delete_oh_data:
+        ohmember.delete_all_files()
+
+    # Delete the actual user
+    User.objects.filter(
+        id=user.id
+    ).delete()
```

Comment:
```suggestion
    user.delete()
```

The existing method of course works too, but one can also delete instances of the class/model by just calling delete on the instance, instead of making another DB query 🙂